src/FFMpeg/Filters/Audio/ImageVideoFilter.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of PHP-FFmpeg.
+ *
+ * (c) Alchemy <info@alchemy.fr>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FFMpeg\Filters\Audio;
+
+use FFMpeg\Filters\TPriorityFilter;
+use FFMpeg\Format\AudioInterface;
+use FFMpeg\Media\Audio;
+
+/**
+ * Converts the audio to video and `artwork` as video content.
+ *
+ * @author     pralhad.shrestha05@gmail.com
+ * @copyright  Jens Hausdorf 2018
+ * @license    MIT License
+ * @package    FFMpeg\Filters
+ * @subpackage Audio
+ */
+class ImageVideoFilter implements AudioFilterInterface
+{
+    use TPriorityFilter;
```

Comment:
I just want to note that I'm impressed and satisfied that you found this trait without any doc.. :)